packages/workbox-webpack-plugin/test/node/test.js

```diff
@@ -61,13 +61,13 @@ describe('Tests for webpack plugin', function() {
 
   it('should mutate config accordin to webpack defaults', () => {
     let swWebpackPlugin = new SwWebpackPlugin({});
-    assert.equal(swWebpackPlugin.getConfig(webpackCompilation).rootDirectory,
+    assert.equal(swWebpackPlugin.getConfig(webpackCompilation).globDirectory,
```

Comment:
Actually this being the unit test, i have stubbed the sw-build details.
I'll open an issue and send the PR for end to end test on monday